Oadp 6250 mod docs - Configuring OADP on AWS

shdeshpa07 opened the pull request:
## Jira 

* [OADP 6250](https://issues.redhat.com/browse/OADP-6250)

This PR is for changes found by running the ASCIIDocDITA vale package for the `Configuring OADP on AWS` doc set. 

This PR has structural changes only. No technical changes have been made.

**Peer/Merge reviewer - Please review the PR from the POV of structural-changes only. The PR has only structural changes. Which means, no content has been newly added/modfied and there are no technical changes. Content is moved around to make it compatible for DITA migration.**

cc: @kalexand-rh 

## Preview

[Configuring OADP on AWS](https://95068--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws)

##  Version

* OCP 4.19, 4.20

## QE Review

By-passing QE review for this PR as the changes are structural only.

shdeshpa07 committed:
OADP-6250-mod-docs-aws

Signed-off-by: Shruti Deshpande <shdeshpa@redhat.com>

anarnold97 reviewed (approved):
LGTM

shdeshpa07 commented:
/label peer-review-needed

shdeshpa07 commented:
/label OADP

xenolinux commented on modules/oadp-using-ca-certificates-with-velero-command.adoc:
```diff
@@ -0,0 +1,77 @@
++
+[source,terminal]
+----
+$ CA_CERT=$(oc -n openshift-adp get dataprotectionapplications.oadp.openshift.io <dpa-name> -o jsonpath='{.spec.backupLocations[0].velero.objectStorage.caCert}')
+
+$ [[ -n $CA_CERT ]] && echo "$CA_CERT" | base64 -d | oc exec -n openshift-adp -i deploy/velero -c velero -- bash -c "cat > /tmp/your-cacert.txt" || echo "DPA BSL has no caCert"
+----
++
+[source,terminal]
+----
+$ velero describe backup <backup_name> --details --cacert /tmp/<your_cacert>.txt
```
Use single command per code block: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#single-command-per-code-block

xenolinux commented on modules/oadp-using-ca-certificates-with-velero-command.adoc:
```diff
@@ -0,0 +1,77 @@
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-using-ca-certificates-with-velero-command-aliased-for-velero-deployment_{context}"]
+== Using CA certificates with the velero command aliased for Velero deployment
```
```suggestion
= Using CA certificates with the velero command alias
```

The first heading should be H1
The heading can be shortened and the intro line can have those details -- Up to you

xenolinux commented on modules/oadp-using-ca-certificates-with-velero-command.adoc:
```diff
@@ -0,0 +1,77 @@
+
+.Procedure
+
+. To use an aliased Velero command, run the following command:
```
```suggestion
. To create an alias for the Velero command, run the following command:
```

Rephrased a little for clarity

xenolinux reviewed (commented):
A few nits before merging; overall looks good

shdeshpa07 commented:
> A few nits before merging; overall looks good

Hi @xenolinux - Thank you for a great review. Actually, the PR is for structural changes only. We are running the ASCIIDocDITA vale package  (shared by Jaromir) to prepare OADP content for DITA migration. So in this PR, I have only moved legacy content around, such as creating new modules/ adding [role=abstract], removing discreet headings, etc.

The new modules that you see are not new content, but existing/legacy content that has been removed from a legacy file and added to a new file. 

I understand, the changes you have suggested make the content better, so we created a [backlog confluence page](https://spaces.redhat.com/display/MMSDOCS/OADP+Mod+Docs+Work) for such items. In this page, we are collecting all such PRs where there are good suggestions but they are out-of-scope for the modularization work. 

We discussed this with @kalexand-rh and Kathryn suggested to create a backlog for such PRs. I am adding this PR too in that list. 

Would you be okay to merge this PR, under the scope of **modularization/structural-changes** only?  I have added a comment in comment 0 too, to review from the POV of modularization. We will get to your suggestions in a future release.

Copying @kalexand-rh for visibility. Thanks.

Thanks!

xenolinux commented:
> > A few nits before merging; overall looks good
> 
> Hi @xenolinux - Thank you for a great review. Actually, the PR is for structural changes only. We are running the ASCIIDocDITA vale package (shared by Jaromir) to prepare OADP content for DITA migration. So in this PR, I have only moved legacy content around, such as creating new modules/ adding [role=abstract], removing discreet headings, etc.
> 
> The new modules that you see are not new content, but existing/legacy content that has been removed from a legacy file and added to a new file.
> 
> I understand, the changes you have suggested make the content better, so we created a [backlog confluence page](https://spaces.redhat.com/display/MMSDOCS/OADP+Mod+Docs+Work) for such items. In this page, we are collecting all such PRs where there are good suggestions but they are out-of-scope for the modularization work.
> 
> We discussed this with @kalexand-rh and Kathryn suggested to create a backlog for such PRs. I am adding this PR too in that list.
> 
> Would you be okay to merge this PR, under the scope of **modularization/structural-changes** only? I have added a comment in comment 0 too, to review from the POV of modularization. We will get to your suggestions in a future release.
> 
> Copying @kalexand-rh for visibility. Thanks.
> 
> Thanks!

Sure sounds good. Merging Thanks

xenolinux commented:
/cherrypick enterprise-4.19

xenolinux commented:
/cherrypick enterprise-4.20
